### ISSUE TITLE
Replace treeify with object-treeify

### ISSUE THREAD

**Issue by simlu** [oclif/cli-ux#88]
I've just spend some time fighting with treeify and then decided to write my own version that is much more concise and efficient.

I'm just curious if you would be interested in replacing [treeify](https://www.npmjs.com/package/treeify) with [object-treeify](https://www.npmjs.com/package/object-treeify). Functionality wise they should be identical and I'd be very happy to create a pull request.

I'm just trying to promote my npm packages a bit more, however I believe that this is an upgrade to code quality and efficiency.

**Comment by RasPhilCo** [comment]
Cool! I think it's important to be able to control sorting. Would you add optional sorting?

**Comment by simlu** [comment]
@RasPhilCo Yes, that would be a very easy addition. I'll take a look later!

**Comment by simlu** [comment]
@RasPhilCo Updated the library. I'm not sure how the sorting would work with console input. However I've created a pull request here for now: https://github.com/oclif/cli-ux/pull/91/files
